### DISCUSSION TITLE
How to change export request headers during run time?

### DISCUSSION THREAD

**Discussion by huliang-microsoft** [open-telemetry/opentelemetry-go#5128] [Q&A] [answered]
Hello, I know that I can set exporter headers using OTEL_EXPORTER_OTLP_HEADERS env vars. But these env vars are used during exporter initialization. Is there a way to set/change the headers during runtime when it is needed?

**Comment by pellared** [comment, marked as answer]
Duplicates https://github.com/open-telemetry/opentelemetry-go/issues/5129
